### PR TITLE
[duplicati] migrate to common

### PR DESCRIPTION
**Pull request by nicholaswilde**
Signed-off-by: Nicholas Wilde <ncwilde43@gmail.com>

**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Comment by nicholaswilde** [comment]
Should I add `influxdb` as a dependency?

File: charts/stable/duplicati/values.yaml [hunk 1]
```diff
@@ -1,166 +1,43 @@
...
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
 
 image:
-  # -- Image repository
   repository: linuxserver/duplicati
-  # -- Image tag. Possible values listed [here](https://hub.docker.com/r/linuxserver/duplicati/tags/).
-  tag: v2.0.5.1-2.0.5.1_beta_2020-01-18-ls72
-  # -- Image pull policy
+  tag: v2.0.5.1-2.0.5.1_beta_2020-01-18-ls98
```
Comment (onedr0p) [inline charts/stable/duplicati/values.yaml:10]: I'd make this latest, it's such a cluster f of a tag name.

File: charts/stable/duplicati/Chart.yaml [hunk 2]
```diff
@@ -2,14 +2,18 @@ apiVersion: v2
 appVersion: v2.0.5.1
```
Comment (onedr0p) [inline charts/stable/duplicati/Chart.yaml:2]: latest here too, for now...

**Review by onedr0p — CHANGES_REQUESTED** [review]

**Comment by onedr0p** [comment]
> Should I add `influxdb` as a dependency?

Nope, it should not be needed.

**Review by onedr0p — APPROVED** [review]

**Pull request merged**